Docs/Benchmarks.md

```diff
@@ -95,17 +95,17 @@ Comparing the Zynq-7000 FPGA accelerated performance to the ARM CPU on the same
 ### Details
 
 - FPGA: Xilinx Zynq XC7Z030-1SBG485C SoC FPGA. Main clock is 150 Mhz.
-- Host: ARM dual-core Cortex-A9 MPCore CPU. Power consumption is around 95 W under load (based on the processor's TDP, [see here](https://ark.intel.com/content/www/us/en/ark/products/64611/intel-xeon-processor-e5-2450-20m-cache-2-10-ghz-8-00-gt-s-intel-qpi.html); the power draw is likely larger when the CPU increases its clock speed under load).
+- Host: ARM dual-core Cortex-A9 MPCore CPU.
 - Both has access to 1 GB (32-bit) DDR3L SDRAM.
-- The power consumption is [unspecified in the official documentation](https://wiki.trenz-electronic.de/display/PD/TE0715+TRM#TE0715TRM-PowerConsumption), but it's less than 21.5W because that's the maximum rating of the power supply we've used. The _Power advantage_ column is omitted due to this lack of information and because of the difficulty of measuring power consumption over such short time spans with the lab equipment we can access at the moment. Assuming max consumption, it would be the same as the _Speed advantage_ columns.
+- Trenz reported the typical power consumption as about 5W. Using an inline mains energy meter we've measured 4.6W minimum. The numbers in the table below show the measured maximums. Two separate builds were repeatedly executed in a loops during measurement. One build only executed the CPU version, while the other only the FPGA version.
 
 ### Measurements
 
-| Algorithm             | Speed advantage |   Parallelism  |    CPU   | CPU power | FPGA utilization | Net FPGA | Total FPGA | FPGA power |
-|:----------------------|:---------------:|:--------------:|:--------:|:---------:|:----------------:|:--------:|:----------:|:----------:|
-| ImageContrastModifier |       4864%     |        25      |  2780 ms |   59.8 Ws |        41%       |   29 ms  |    56 ms   |   1.2 Ws   |
-| MonteCarloPiEstimator |       8646%     |        77      |  3236 ms |   69.6 Ws |        33%       |   31 ms  |    37 ms   |   0.8 Ws   |
-| ParallelAlgorithm     |       30582%    |       260      | 66273 ms | 1424.9 Ws |        55%       |  210 ms  |   216 ms   |   4.6  Ws  |
+| Algorithm             | Speed advantage | Power advantage |   Parallelism  |    CPU   | CPU watts | CPU power | FPGA utilization | Net FPGA | Total FPGA | FPGA watts | FPGA power |

```

Comment:
Az FPGA watts az ugyanúgy a hálózati mérést? Mert abban ugye a CPU is benne van.

Reply:
Igen, mindkettő hálózati. Írjam át mondjuk "FPGA + CPU watts"-ra vagy valami hasonló? (vagy meg lehetne csillagozni mindkettőt és a táblázat alá írni, hogy "System watts with CPU payload" és "System watts with FPGA payload")
Nincs ötletem, hogyan lehetne csak az FPGA (vagy ha már itt tartunk csak tisztán CPU) mérést csinálni. Legalábbis nincs olyan ötletem ami nem lenne veszélyes a hardverre. :D

Reply:
Hozzáadtam így:
```
1. Total system watts with CPU payload.
2. Total system watts with FPGA payload.
```

Reply:
Aha, úgy jó lesz.